admin/templates/configuration/tmp/configure/sql/qserv-worker.sql

```diff
@@ -1,13 +1,42 @@
 CREATE USER IF NOT EXISTS '{{MYSQLD_USER_QSERV}}'@'localhost';
 
 -- Used by xrootd Qserv plugin:
--- to publish LSST databases
+-- to publish LSST databases, tables and chunks
+
 DROP DATABASE IF EXISTS qservw_worker;
 CREATE DATABASE qservw_worker;
+
 GRANT SELECT ON qservw_worker.* TO '{{MYSQLD_USER_QSERV}}'@'localhost';
+
 CREATE TABLE qservw_worker.Dbs (
-  `db` char(200) NOT NULL
-);
+
+  `db` CHAR(200) NOT NULL,
+
+  PRIMARY KEY (`db`)
+
+) ENGINE=InnoDB;
```

Comment:
I thought about this too and decided to leave the old table for backward compatibility with the existing code which may be run against the new instances of the worker databases. The old table could be removed later when the transition will finish.

Reply:
Another reason why me keep the old table is to allow enabling/disabling whole databases w/o destroying chunk maps. Though, this could (or should?) be done on the czar.